scripts/rebase-ledger/index.js

```diff
@@ -0,0 +1,6 @@
+const { rebase } = require("./rebase-ledger.js");
```

Comment:
Does this file serve any immediate purpose? If not why merge it? (This isn't a rhetorical question, I imagine there might be a reason)

Reply:
Separation of concerns. It lets the `rebase` module exist independent of usage/environment, letting this file parse CLI arguments and output the result. Logic in one file, IO in another.